specification/resources/gen-ai/GenAIAPI_CreateAgentAPIKeyPublic.yml

```
@@ -0,0 +1,39 @@
+description: To create an agent API key, send a POST request to `/v2/gen-ai/agents/{agent_uuid}/api_keys`.
+operationId: Create_an_Agent_API_Key
```

```suggestion
operationId: genAI_agent_api_key_create
```

Can we follow the naming practice used in the rest of the project for these? When generating API clients from the OpenAPI spec (e.g. https://github.com/digitalocean/pydo), the `operationId` is used for the method names.

They should be in the format of `<namespace>_<operation>`.  For example: [`sshKeys_list`](https://github.com/digitalocean/openapi/blob/dc4c36f5568121f2502da2a9a7f32b4d63ec41ba/specification/resources/ssh_keys/sshKeys_list.yml#L1) becomes [`ssh_keys.list()`](https://docs.digitalocean.com/reference/pydo/reference/ssh_keys/list/).

So for this one example, we probably want something more like `genAI_agent_api_key_create`

---

amanda requested the operation id be the summary. is it enough to add the genAi_ in front of it?